pysass.cpp

```diff
@@ -535,6 +540,18 @@ PySass_compile_string(PyObject *self, PyObject *args) {
     sass_option_set_include_path(options, include_paths);
     sass_option_set_precision(options, precision);
     sass_option_set_is_indented_syntax_src(options, indented);
+    sass_option_set_source_map_contents(options, source_map_contents);
+    sass_option_set_source_map_embed(options, source_map_embed);
+    sass_option_set_omit_source_map_url(options, omit_source_map_url);
+
+    if (PyBytes_Check(source_map_root)) {
+        if (PyBytes_GET_SIZE(source_map_root)) {
```

Comment:
combine these two if statements w/ `&&`

Reply:
alternatively, we could enforce this to always be a bytestring in the python code and only check the length here 🤷‍♂️